relay-genesis/mock-difficulty.js

```diff
@@ -0,0 +1,23 @@
+const BitcoinRpc = require('bitcoind-rpc')
+const Bluebird = require('bluebird')
+const config = require("../configs/bitcoin/config.json")
+
+const MockRelay = artifacts.require('MockRelay.sol');
+
+const bitcoinRpc = new BitcoinRpc(config)
+Bluebird.promisifyAll(bitcoinRpc)
+
+module.exports = async function() {
+    try {
+        const difficulty = new web3.utils.BN((await bitcoinRpc.getdifficultyAsync()).result)
+
+        console.log(`chain difficulty is ${difficulty}`)
+
+        const mockRelay = await MockRelay.deployed()
+
+        await mockRelay.setCurrentEpochDifficulty(difficulty)
+        await mockRelay.setPrevEpochDifficulty(difficulty)
```

Comment:
Maybe we could get `CurrentEpochDifficulty` and use it in `setPrevEpochDifficulty` instead of setting both current and previous epoch difficulty to the current chain value?

Reply:
It's not valid regarding the context we use this script. The difficulty of the private BTC chain is constant and is equal to `~0.4`. Default values in `MockRelay.sol` are `1` so if we use the current difficulty to set the previous one at the beginning we will end up with:
- `prevEpochDifficulty`: `1`
- `currentEpochDifficulty` : `~0.4`
which is not valid because previous difficulty for our chain was also `~0.4` not `1`.